components/grade-table.js

```diff
@@ -0,0 +1,25 @@
+class GradeTable {
+  constructor(tableElement){
+      this.tableElement = tableElement;
+  }
+  updateGrades(grades){
+    let updateTableElement = this.tableElement.querySelector('tbody');
+    updateTableElement.textContent = "";
+
+    for(let i=0; i < grades.length; i++) {
+      let tr   = document.createElement('tr');
+
+      let tdName   = document.createElement('td');
+      let tdCourse = document.createElement('td');
+      let tdGrade  = document.createElement('td');
```

Comment:
If you are going to use let/const then these ought to be const (since they arent being changed directly) otherwise feel free to use var!